src/ConfigurationsProvider.tsx

```diff
@@ -49,13 +65,23 @@ export const ConfigurationsProvider = ({ children }: { children?: ReactNode }) =
     skip: !Object.keys(sue).length
   });
 
+  const { data: filterData } = useQueryWithApollo(getQuery(QUERY_TYPES.RESOURCE_FILTERS), {
+    skip: !showResourceFilters
+  });
+
   const mergedConfig = useMemo(() => {
-    if (!Object.keys(sue).length) {
+    if (!Object.keys(sue).length || !showResourceFilters) {
       return defaultConfiguration;
     }
 
+    const resourceFilters = filterData?.resourceFilters?.map((filter: any) => ({
+      ...filter,
+      dataResourceType: FILTER_QUERY_TYPES[filter.dataResourceType]
+    }));
```

Comment:
i think there is something missing with checking if everything is not active, where we could `return defaultConfiguration` instead of the current `!showResourceFilters`?

Reply:
If the `showResourceFilters` value in `globalSettings` is missing or false, the `defaultConfiguration`, `resourceFilters` if exists, and other config

Reply:
I sent a new commit. Can you please check again?

https://github.com/smart-village-solutions/smart-village-app-app/pull/1188/commits/676bf433e9863a752a1b59a722cb4e6658c7ac4a